buku

```diff
@@ -6071,7 +6071,7 @@ POSITIONAL ARGUMENTS:
     addarg('--np', action='store_true', help=hide)
     addarg('-o', '--open', nargs='*', help=hide)
     addarg('--oa', action='store_true', help=hide)
-    addarg('--default-scheme', nargs='1', default=SCHEME_HTTP, help=hide)
+    addarg('--default-scheme', nargs='?', default=SCHEME_HTTP, help=hide)
```

Comment:
`?` [means](https://docs.python.org/3/library/argparse.html#nargs) it's valid to use `--default-scheme` without providing an argument. What's the usecase for that? :thinking: